src/core/sentences/global-sentences.storage.ts

```diff
@@ -0,0 +1,29 @@
+import { SentenceEntityGlobal } from './global-sentences.model';
+
+export const defaultSentences: SentenceEntityGlobal[] = [
+  {
+    verb: 'win',
+    sentence: `Brazil _______ five world cups, it’s incredible!`,
+    rigthAnswer: 'won',
+  },
+  {
+    verb: 'win',
+    sentence: `I only want to see whether you will let me _______ this ten, or beat it.`,
```

Comment:
Same with the rest of sentneces

```
I only want to see whether you will let me {verb} this ten, or beat it.
```

We will do a replace once we want to show the sentence

Reply:
done